bff/src/Bff/Endpoints/DefaultUserService.cs

```diff
@@ -3,22 +3,21 @@
 
 using System.Text;
 using System.Text.Json;
+using Duende.Bff.Configuration;
 using Duende.Bff.Internal;
 using Duende.IdentityModel;
 using Microsoft.AspNetCore.Authentication;
-using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Microsoft.IdentityModel.Protocols.OpenIdConnect;
 
-// ReSharper disable once CheckNamespace
-namespace Duende.Bff;
+namespace Duende.Bff.Endpoints;
 
 /// <summary>
 /// Service for handling user requests
 /// </summary>
-public class DefaultUserService(IOptions<BffOptions> options, ILoggerFactory loggerFactory) : IUserService
+internal class DefaultUserService(IOptions<BffOptions> options, ILoggerFactory loggerFactory) : IUserService
```

Comment:
When we make this internal, we should add an extensibility point for customizing how management and user claims are retrieved. (Okay to do as a follow up.)